knora-ontologies/knora-base.ttl

```diff
@@ -1473,7 +1459,7 @@
 
 ###  http://www.knora.org/ontology/knora-base#valueHas
 
-:valueHas rdf:type owl:DatatypeProperty ;
+:valueHas rdf:type rdf:Property ;
```

Comment:
Why would a subproperty of `valueHas` be an object property? Wouldn't it rather be a subproperty of `hasLinkTo` then?